.gitlab-ci.yml

```diff
@@ -22,22 +19,18 @@ variables:
 .build:                            &docker_build
   stage:                           build
   image:                           docker:stable
-  only:
-    refs:
-      - schedules
-      - web
-    kubernetes:                    active
```

Comment:
@gabreal FYI, I'm removing this since it is about if k8s is turned on for the project, it doesn't check the status of k8s. And "rules:" syntax does not have this status implemented.